### PR TITLE
hardcoded paths are modified

### PR DESCRIPTION
**Pull request by KiSooLee**
Fixed paths are removed and new path to find CLHEP is applied

### CODE REVIEW

**Comment by KiSooLee** [comment]
+1

**Comment by davidlt** [comment]
There are another 6 lines where `SIMPATH` is used, please, fix those lines to use `$ENV{CLHEP_PARAM_PATH}`.

File: cmake_modules/FindCLHEP.cmake [hunk 1]
```diff
@@ -45,20 +42,20 @@ If(CLHEP_CONFIG_EXE)
...
-		${SIMPATH}/include
+		$ENV{CLHEP_PARAM_PATH}/cern/clhep/include
+		$ENV{CLHEP_PARAM_PATH}/include
 		NO_DEFAULT_PATH
 	)
 
 	FIND_PATH(CLHEP_BASE_DIR NAMES ClhepVersion.h PATHS
-		${SIMPATH}/cern/clhep
-		${SIMPATH}/include
+		$ENV{CLHEP_PARAM_PTH}/cern/clhep
```
Comment (davidlt) [inline cmake_modules/FindCLHEP.cmake]: typo here, please fix.

**Comment by davidlt** [comment]
And squash all the commits into a single.

**Comment by KiSooLee** [comment]
squash is done

**Comment by davidlt** [comment]
LGTM. Hope you did local tests to make sure it works.

**Pull request merged**